src/main/scala/org/broadinstitute/dsde/firecloud/webservice/EntityApiService.scala

```diff
@@ -40,18 +42,19 @@ trait EntityService extends HttpService with PerRequestCreator with FireCloudDir
             } ~
               path("copy") {
                 post {
-                  requireUserInfo() { _ =>
+                  requireUserInfo() { userInfo =>
                     parameter('linkExistingEntities.?) { linkExistingEntities =>
                       entity(as[EntityCopyWithoutDestinationDefinition]) { copyRequest =>
                         val linkExistingEntitiesBool = Try(linkExistingEntities.getOrElse("false").toBoolean).getOrElse(false)
-                        requestContext =>
                           val copyMethodConfig = new EntityCopyDefinition(
                             sourceWorkspace = copyRequest.sourceWorkspace,
                             destinationWorkspace = WorkspaceName(workspaceNamespace, workspaceName),
                             entityType = copyRequest.entityType,
                             entityNames = copyRequest.entityNames)
                           val extReq = Post(FireCloudConfig.Rawls.workspacesEntitiesCopyUrl(linkExistingEntitiesBool), copyMethodConfig)
-                          externalHttpPerRequest(requestContext, extReq)
+
+
+                          complete { executeRequestRaw(userInfo.accessToken)(extReq) }
```

Comment:
i don't like executing these requests right from here. will refactor a bit more

Reply:
actually i will leave whether or not i do this refactor up to the reviewers. i think this PR has moved orch in a better direction in terms of standardizing how we make external requests and separate our service layers, but i am doubting that i need to do all of it right now in this PR. this could easily be followup cleanup and make for a more manageable review...